exercises/practice/gigasecond/.docs/instructions.append.md

```diff
@@ -1,3 +1,6 @@
 # Instructions append
 
 It is possible to return a correct value for this exercise by mutating the solution function argument. Although there are legitimate use cases for mutating function arguments, this is usually undesirable, and in the case of this exercise, clearly unexpected. For this reason, the test suite has a test that fails in case the argument has been modified after the function execution.
+
+## Notes
+The input and output of the gigasecond function is of type [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date).
```

Comment:
* Put your two new lines before line 3. Line 3 is also a note but yours is more important.
* Wrap `gigasecond` in backticks, see https://exercism.org/docs/building/markdown/markdown#h-code
* See https://exercism.org/docs/building/markdown/markdown#h-links for how we provide links in the markdown

Currently, the CI pipeline fails because the formatting is not right. See https://github.com/exercism/javascript/blob/main/CONTRIBUTING.md#format for how to run the formatter.